src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java

```diff
@@ -0,0 +1,34 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.Location;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.EntityType;
+
+import javax.annotation.Nonnull;
+
+public class ArmorStandUtils {
+
+    /**
+     * This method returns a template {@link ArmorStand} that has all of its
+     * values set.
+     *
+     * @param l
+     *            The {@link Location} to spawn the Armor Stand at
+     * @return the {@link ArmorStand}
+     */
+    public static @Nonnull ArmorStand spawnArmorStand(@Nonnull Location location) {
```

Comment:
I wouldnt call this specifically spawnarmorstand since you give it a bunch of attributes

Reply:
So I call it that due to those values being used for almost Every Armor Stand in Slimefun, however I'll change it to be called something like Phony Hologram because its not registered as a official armor stand

Reply:
as a official hologram*

Reply:
latest commit